_security-plugin/configuration/security-admin.md

```diff
@@ -299,3 +302,30 @@ Name | Description
 `-era` | Enable replica auto-expand.
 `-dra` | Disable replica auto-expand.
 `-us` | Update the replica settings.
+
+## Windows usage
+
+On Windows, the equivalent of `securityadmin.sh` is the `securityadmin.bat` script located in the `path/to/opensearch-{{site.opensearch_version}}/plugins/opensearch-security/tools/` directory.
+
+When running the example commands in the preceding sections, use the **command prompt**. You can open the command prompt by entering `cmd` in the search box next to **Start** on the taskbar. 
+
+Note that on Windows you need the `call` command to call a batch script. 
```

Comment:
Is `call` necessary? From what I am reading it looks like `call` was introduced to call batch scripts within a batch script. 

Reference: https://stackoverflow.com/a/14732373

Reply:
I tried without `call` and it didn't work: it said that .\ is not recognized as a command.

Reply:
Interesting. Does it work without the `.\`? i.e. `plugins/opensearch-security/tools/securityadmin.bat` instead of `call ./plugins/opensearch-security/tools/securityadmin.bat`

The official documentation from MS seems to indicate that `call` does not have any effect outside of a script or batch file: https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/call

>  Note

> Call has no effect at the command prompt when it is used outside of a script or batch file.

I bring this up because if we can use the same set of commands on both `cmd` and `Powershell` then it simplifies the documentation.

Reply:
I just tried with without call on `cmd` on `Windows Server 2016 Datacenter` and it ran the script. 

Running: `.\plugins\opensearch-security\tools\install_demo_configuration.bat` within the OpenSearch home directory.

Reply:
@cwperks thank you for testing and confirming! It must be me then. I updated the docs.

Reply:
Last thing to make sure the same command works on both `cmd` and `powershell`. In my testing it looks like `cmd` requires backslashes while powershell can work with either. I think we should make all of these windows commands use backslashes so that it works in both terminals.

Reply:
Interesting. I'm on Windows 10, and both forward slashes and backslashes work in command prompt. In any case, I have updated the docs to have backslashes only. Thank you!